x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/index.tsx

```diff
@@ -73,7 +74,7 @@ export class Waterfall extends Component<Props, State> {
           color={serviceColors[item.serviceName]}
           item={item}
           totalDuration={waterfall.duration}
-          isSelected={true} // TODO: implement logic
+          isSelected={true} // TODO: implement logic (do we still need this if state is handled via query param?)
```

Comment:
This is for highlighting the selected row (the one for which the flyout is open).